### DISCUSSION TITLE
V3 firmware upper limit?

### DISCUSSION THREAD

**Discussion by madjetey** [gtxaspec/wz_mini_hacks#403] [Q&A]
Got a pair of V3s running 4.36.8.15.  Despite wyze asking, I've not updated either firmware however mini_hacks doesn't seem to be taking effect.  Can't connect to ssh on either camera as I get a putty refuse error on port 22.  Is there a recommended firmware the V3 should be running?  Only indication I've seen to that is another thread talking of downgrading to 4.36.2.5 for to help avoid wyze app / cloud / account altogether.  I've downloaded it but not pulled the trigger on the downgrade process yet.

**Comment by bjs-pdx** [comment]
From the read me file:

`Works on ANY firmware release up to 4.36.9.139 (DO NOT UPGRADE BEYOND THIS FW IF YOU WANT RTSP or any cmd FEATURES!) `

**Reply by madjetey** [reply]
Will never work on hardware with 3 hrs of sleep ever again

**Comment by virmaior** [comment]
>  I've not updated either firmware however mini_hacks doesn't seem to be taking effect. 

No message "Wz mini initiating first boot?"
If not, then a quite likely cause is that your SD card is not FAT32 and so the hack is never running.

> Can't connect to ssh on either camera as I get a putty refuse error on port 22. 
That should work on any version if the hack is actually running.
It's quite the bear setting this up the first time because it's using ssh authorized keys.

> Is there a recommended firmware the V3 should be running? 

I think the recommend one is to run the 4.36.9.139 version.

> Only indication I've seen to that is another thread talking of downgrading to 4.36.2.5 for to help avoid wyze app / cloud / account altogether. 

I'm on this 4.36.2.5. The main issue *was* that higher versions have more phone home code and *couldn't operate* without internet. There's now a switch that can fix this.

**Reply by madjetey** [reply]
Checked.  Doubled checked.  Used different microSDs all formatted to Fat32.  Nothing.

**Comment by madjetey** [comment]
Flashing 4.36.9.139 worked a treat and mini_hacks runs just fine... on one of the 2 cameras. Looks like that one's suffering from a bad microSD card slot and won't read any card at all. Time to work on amazon trans-border returns.
